src/Api/Client.php

```diff
@@ -346,7 +346,7 @@ protected function _arrayToXml(array $array, SimpleXMLElement $xml, $parentEl =
             if (is_array($value)) {
                 $this->_arrayToXml($value, $this->_isAssocArray($value) ? $xml->addChild($el) : $xml, $el);
             } else {
-                $xml->addChild($el, $value);
+                $xml->addChild($el, htmlspecialchars($value));
```

Comment:
why not `$xml->{$el} = $value`?

Reply:
then we get the problem
_It is not yet possible to assign complex types to properties_

Reply:
hmm, does it work with `htmlspecialchars`? It seems to cast string type for any complex type.